docs/polyglot/python.md

```diff
@@ -19,6 +19,15 @@ Python in the runtime. Please familiarise yourself with the general operation of
 
 <!-- /MarkdownTOC -->
 
+## Install Graal Python standalone distribution
+
+Before you can use Python in Enso you need to install Graal Python standalone
```

Comment:
People can use Python without installing Graal Python... just to follow this tutorial one needs to install standalone distribution.

Reply:
Good point. Instructions updated at 758ffcf5589db25741ac38db9ef4316224c7bb1b